lua/colorful-menu/init.lua

```diff
@@ -599,6 +601,50 @@ function M.adjust_range(item, name_offset, label)
 	return item
 end
 
+function M.php_intelephense_compute_completion_highlights(completion_item, ft)
+    local label = completion_item.label
+    local detail = completion_item.labelDetails and completion_item.labelDetails.detail or completion_item.detail
+    local kind = completion_item.kind
+
+    if (kind == M.Kind.Function or kind == M.Kind.Method) and #detail > 0 then
+        local signature = detail:sub(#label+1)
+        local text = string.format("%s <?php fn %s {}", label, signature)
+        local item = M.highlight_range(text, ft, 6 + #label, #text - 3)
+        return M.adjust_range(item, #label+1, label)
+    elseif kind == M.Kind.EnumMember and #detail > 0 then
+        local text = string.format("%s <?php %s;", label, detail)
+        local item = M.highlight_range(text, ft, #label + 6, #text - 2)
```

Comment:
You need to adjust to remove the last `;`
```suggestion
        local item = M.highlight_range(text, ft, #label + 6, #text - 1)
```